shortstories/templates/shortstories/shortstories_edit.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base/index.html' %}
+
+{% block content %}
+    <h1>New Short Story</h1>
+    <form method="POST" class="post-form">{% csrf_token %}
+        {{ form.as_p }}
```

Comment:
Do you show the error messages here? i.e. missing fields or blank status, etc...